### ISSUE TITLE
[Support]: Camera keeps failing on built in go2rtc inside frigate 0.12 only if viewing webrtc

### ISSUE THREAD

**Issue by Uriziel01** [blakeblackshear/frigate#6151]
### Describe the problem you are having

TL;DR; - one of my cameras inside Frigate stops working sometimes when I try to access the webrtc stream, only way to fix it is to restart Frigate addon. Webrtc works reliable and with no issues in standalone go2rtc outside Frigate.

My doorbell was working really stable for the longest time in Frigate using just the jsmpeg stream as a live view, but after migrating to 0.12 I'm trying to use the go2rtc component inside Frigate and when I try to use webrtc live view my camera sometimes just stops responsing in Frigate UI and all I get is `No frames have been received, check error logs` screen. As long as I only use jsmpeg or just don't enter live view it's ok, as soon as I try to open the webrtc stream it's a lottery, sometimes it works, sometime it break the camera, even when it loads webrtc it takes unusually long time to do so. Only way to get it back to working is to do a full restart of Frigate.

**IMPORTANT NOTES:**

- I have 4 cameras in total, other 3 works without any problems.
- If I install go2rtc as a standalone hassio addon I can load the webrtc stream dozens of times, it never fails. But then I'm again limited to jsmpeg inside of Frigate.

### Version

0.12.0-da3e197

### Frigate config file

```yaml
birdseye:
  enabled: True
  mode: continuous
mqtt:
  host: XXX
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  user: XXX
  password: "XXX"
  stats_interval: 60
go2rtc:
  streams:
    garden:
      - "rtsp://XXX/user=XXX_password=XXX_channel=0_stream=0.sdp"
    garden_sub:
      - "rtsp://XXX/user=XXX_password=XXX_channel=0_stream=1.sdp"
    doorbell:
      - "rtsp://XXX/onvif1"
    garage:
      - "rtsp://XXX/user=XXX_password=XXX_channel=1_stream=0.sdp"
    right_room:
      - "rtsp://XXX/stream1"
detectors:
  ov:
    type: openvino
    device: AUTO
    model:
      path: /openvino-model/ssdlite_mobilenet_v2.xml
model:
  width: 300
  height: 300
  input_tensor: nhwc
  input_pixel_format: bgr
  labelmap_path: /openvino-model/coco_91cl_bkgr.txt
cameras:
  doorbell:
    live:
      stream_name: doorbell
    mqtt:
      enabled: True
      bounding_box: True
      crop: True
    ffmpeg:
      input_args:
        - -avoid_negative_ts
        - make_zero
        - -fflags
        - nobuffer
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts+discardcorrupt
        - -use_wallclock_as_timestamps
        - "1"
      inputs:
        - path: rtsp://127.0.0.1:8554/doorbell
          input_args: preset-rtsp-restream
          roles:
            - detect
            - record
    objects:
      track:
        - person
    detect:
        enabled: True
        width: 1920
        height: 1080
        fps: 10
    record:
      enabled: True
      retain:
        days: 2
      events:
        retain:
          default: 1
          mode: motion
  garage:
    live:
      stream_name: garage
    mqtt:
      enabled: True
      bounding_box: True
      crop: True
    ffmpeg:
      input_args:
        - -avoid_negative_ts
        - make_zero
        - -fflags
        - nobuffer
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts+discardcorrupt
        - -use_wallclock_as_timestamps
        - "1"
      inputs:
        - path: rtsp://127.0.0.1:8554/garage
          input_args: preset-rtsp-restream
          roles:
            - detect
            - record
    detect:
        width: 1920
        height: 1080
        fps: 10
    objects:
      track:
        - person
      mask:
        - 394,77,394,0,0,0,0,154
        - 1920,0,1920,411,1347,123,1363,0
    record:
      enabled: True
      retain:
        days: 2
      events:
        retain:
          mode: motion
          default: 7
  garden:
    live:
      stream_name: garden
    mqtt:
      enabled: True
      bounding_box: True
      crop: True
    ffmpeg:
      output_args:
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy
      input_args:
        - -avoid_negative_ts
        - make_zero
        - -fflags
        - nobuffer
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts+discardcorrupt
        - -use_wallclock_as_timestamps
        - "1"
      inputs:
        - path: rtsp://127.0.0.1:8554/garden_sub
          input_args: preset-rtsp-restream
          roles:
            - detect
            - record
    objects:
      track:
        - person
      mask: 704,0,704,0,704,140,434,176,372,146,63,172,0,352,0,0
    record:
      enabled: True
      retain:
        days: 2
      events:
        retain:
          mode: motion
          default: 7
    detect:
        width: 704
        height: 576
        fps: 10
  right_room:
    live:
      stream_name: right_room
    mqtt:
      enabled: True
      bounding_box: True
      crop: True
    ffmpeg:
      input_args: preset-rtsp-generic
      inputs:
        - path: rtsp://127.0.0.1:8554/right_room
          input_args: preset-rtsp-restream
    detect:
        width: 1920
        height: 1080
        fps: 10
    objects:
      track:
        - person
ffmpeg:
  global_args: -hide_banner -loglevel info -threads 2 -timeout 90000000
  hwaccel_args: preset-vaapi
  output_args:
    record: preset-record-generic
```

### Relevant log output

```shell
2023-04-18 21:40:12.840996142  172.30.32.1 - - [18/Apr/2023:23:40:12 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:40:15.730990142  172.30.32.2 - - [18/Apr/2023:23:40:15 +0200] "GET /live/webrtc/api/ws?src=doorbell HTTP/1.1" 101 4 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:16.872621715  [2023-04-18 23:40:16] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:40:16.872894612  [2023-04-18 23:40:16] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:40:20.015658622  172.30.32.2 - - [18/Apr/2023:23:40:20 +0200] "GET /live/jsmpeg/doorbell HTTP/1.1" 101 0 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:20.810578853  172.30.32.1 - - [18/Apr/2023:23:40:20 +0200] "GET /api/stats HTTP/1.1" 200 2937 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:40:20.946189320  172.30.32.2 - - [18/Apr/2023:23:40:20 +0200] "GET /live/webrtc/api/ws?src=doorbell HTTP/1.1" 101 4 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:20.947862375  172.30.32.2 - - [18/Apr/2023:23:40:20 +0200] "GET /ws HTTP/1.1" 101 4096 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:20.977544272  172.30.32.2 - - [18/Apr/2023:23:40:20 +0200] "GET // HTTP/1.1" 200 6701 "https://<MY_PUBLIC_DOMAIN_AND_PORT>/ccab4aaf_frigate" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:21.007211095  172.30.32.2 - - [18/Apr/2023:23:40:21 +0200] "GET /api/config HTTP/1.1" 200 13231 "https://<MY_PUBLIC_DOMAIN_AND_PORT>/api/hassio_ingress/77DRobv3MRvNbL2PqHnDV55ihi4cPDggJYjyyYjUZ-s//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:21.113481684  172.30.32.2 - - [18/Apr/2023:23:40:21 +0200] "GET //api/right_room/latest.jpg?h=275 HTTP/1.1" 200 19188 "https://<MY_PUBLIC_DOMAIN_AND_PORT>/api/hassio_ingress/77DRobv3MRvNbL2PqHnDV55ihi4cPDggJYjyyYjUZ-s//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:21.118437023  172.30.32.2 - - [18/Apr/2023:23:40:21 +0200] "GET //api/garden/latest.jpg?h=401 HTTP/1.1" 200 12644 "https://<MY_PUBLIC_DOMAIN_AND_PORT>/api/hassio_ingress/77DRobv3MRvNbL2PqHnDV55ihi4cPDggJYjyyYjUZ-s//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:21.122735380  172.30.32.2 - - [18/Apr/2023:23:40:21 +0200] "GET //api/garage/latest.jpg?h=275 HTTP/1.1" 200 11612 "https://<MY_PUBLIC_DOMAIN_AND_PORT>/api/hassio_ingress/77DRobv3MRvNbL2PqHnDV55ihi4cPDggJYjyyYjUZ-s//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:21.129372784  172.30.32.2 - - [18/Apr/2023:23:40:21 +0200] "GET //api/doorbell/latest.jpg?h=275 HTTP/1.1" 200 6801 "https://<MY_PUBLIC_DOMAIN_AND_PORT>/api/hassio_ingress/77DRobv3MRvNbL2PqHnDV55ihi4cPDggJYjyyYjUZ-s//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:21.759953727  [2023-04-18 23:40:21] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:40:21.759958875  [2023-04-18 23:40:21] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:40:21.759960723  [2023-04-18 23:40:21] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:40:26.882706023  [2023-04-18 23:40:26] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:40:26.882709974  [2023-04-18 23:40:26] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:40:28.819282329  172.30.32.1 - - [18/Apr/2023:23:40:28 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:40:30.222559028  172.30.32.2 - - [18/Apr/2023:23:40:30 +0200] "GET /api/config HTTP/1.1" 200 13231 "https://<MY_PUBLIC_DOMAIN_AND_PORT>/api/hassio_ingress/77DRobv3MRvNbL2PqHnDV55ihi4cPDggJYjyyYjUZ-s//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:30.728627769  23:40:30.728 WRN [streams] json locked
2023-04-18 21:40:31.774330772  [2023-04-18 23:40:31] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:40:31.774597960  [2023-04-18 23:40:31] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:40:31.774600713  [2023-04-18 23:40:31] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:40:33.017055507  172.30.32.2 - - [18/Apr/2023:23:40:33 +0200] "GET //api/stats HTTP/1.1" 200 2998 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:40:36.826106427  172.30.32.1 - - [18/Apr/2023:23:40:36 +0200] "GET /api/stats HTTP/1.1" 200 2968 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:40:36.895106919  [2023-04-18 23:40:36] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:40:36.895546749  [2023-04-18 23:40:36] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:40:41.780800957  [2023-04-18 23:40:41] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:40:41.781055309  [2023-04-18 23:40:41] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:40:41.781060597  [2023-04-18 23:40:41] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:40:44.812963200  172.30.32.1 - - [18/Apr/2023:23:40:44 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:40:46.909425345  [2023-04-18 23:40:46] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:40:46.910451195  [2023-04-18 23:40:46] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:40:51.791568010  [2023-04-18 23:40:51] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:40:51.791743955  [2023-04-18 23:40:51] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:40:51.791746521  [2023-04-18 23:40:51] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:40:52.830075685  172.30.32.1 - - [18/Apr/2023:23:40:52 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:40:56.916744628  [2023-04-18 23:40:56] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:40:56.918417979  [2023-04-18 23:40:56] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:41:00.755961163  23:41:00.755 WRN [streams] json locked
2023-04-18 21:41:00.811563367  172.30.32.1 - - [18/Apr/2023:23:41:00 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:41:01.800094702  [2023-04-18 23:41:01] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:41:01.800700813  [2023-04-18 23:41:01] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:41:01.800703621  [2023-04-18 23:41:01] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:41:05.744698011  172.30.32.2 - - [18/Apr/2023:23:41:05 +0200] "GET /api/config HTTP/1.1" 200 13231 "https://<MY_PUBLIC_DOMAIN_AND_PORT>/api/hassio_ingress/77DRobv3MRvNbL2PqHnDV55ihi4cPDggJYjyyYjUZ-s//" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:41:06.921254542  [2023-04-18 23:41:06] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:41:06.921316903  [2023-04-18 23:41:06] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:41:08.812855381  172.30.32.1 - - [18/Apr/2023:23:41:08 +0200] "GET /api/stats HTTP/1.1" 200 2967 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:41:11.815221509  [2023-04-18 23:41:11] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:41:11.815779306  [2023-04-18 23:41:11] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:41:11.816292378  [2023-04-18 23:41:11] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:41:16.780843161  172.30.32.1 - - [18/Apr/2023:23:41:16 +0200] "GET /api/stats HTTP/1.1" 200 2965 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:41:16.948852846  [2023-04-18 23:41:16] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:41:16.949061408  [2023-04-18 23:41:16] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:41:21.825331565  [2023-04-18 23:41:21] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:41:21.826888343  [2023-04-18 23:41:21] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:41:21.826897480  [2023-04-18 23:41:21] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:41:24.817810337  172.30.32.1 - - [18/Apr/2023:23:41:24 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:41:26.975061198  [2023-04-18 23:41:26] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:41:26.975127973  [2023-04-18 23:41:26] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:41:30.791775494  23:41:30.791 WRN [streams] json locked
2023-04-18 21:41:31.843481839  [2023-04-18 23:41:31] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:41:31.843658298  [2023-04-18 23:41:31] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:41:31.846735069  [2023-04-18 23:41:31] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:41:32.811529648  172.30.32.1 - - [18/Apr/2023:23:41:32 +0200] "GET /api/stats HTTP/1.1" 200 2999 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:41:36.970490090  [2023-04-18 23:41:36] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:41:36.971158636  [2023-04-18 23:41:36] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:41:40.841216092  172.30.32.1 - - [18/Apr/2023:23:41:40 +0200] "GET /api/stats HTTP/1.1" 200 2937 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:41:41.852039642  [2023-04-18 23:41:41] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:41:41.852458795  [2023-04-18 23:41:41] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:41:41.852467081  [2023-04-18 23:41:41] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:41:46.995369101  [2023-04-18 23:41:46] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:41:46.995690375  [2023-04-18 23:41:46] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:41:48.829800803  172.30.32.1 - - [18/Apr/2023:23:41:48 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:41:51.872415379  [2023-04-18 23:41:51] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:41:51.879928941  [2023-04-18 23:41:51] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:41:51.879938480  [2023-04-18 23:41:51] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:41:56.827966560  172.30.32.1 - - [18/Apr/2023:23:41:56 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:41:57.012846097  [2023-04-18 23:41:57] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:41:57.012939999  [2023-04-18 23:41:57] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:42:00.808231651  23:42:00.808 WRN [streams] json locked
2023-04-18 21:42:01.894748175  [2023-04-18 23:42:01] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:42:01.894752231  [2023-04-18 23:42:01] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:42:01.894754009  [2023-04-18 23:42:01] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:42:04.827067026  172.30.32.1 - - [18/Apr/2023:23:42:04 +0200] "GET /api/stats HTTP/1.1" 200 2965 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:42:07.019305935  [2023-04-18 23:42:07] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:42:07.019501601  [2023-04-18 23:42:07] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:42:11.909462280  [2023-04-18 23:42:11] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:42:11.910181674  [2023-04-18 23:42:11] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:42:11.910184911  [2023-04-18 23:42:11] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
2023-04-18 21:42:12.835618403  172.30.32.1 - - [18/Apr/2023:23:42:12 +0200] "GET /api/stats HTTP/1.1" 200 2969 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:42:17.034541396  [2023-04-18 23:42:17] frigate.video                  ERROR   : doorbell: Unable to read frames from ffmpeg process.
2023-04-18 21:42:17.035052060  [2023-04-18 23:42:17] frigate.video                  ERROR   : doorbell: ffmpeg process is not running. exiting capture thread...
2023-04-18 21:42:20.563831324  172.30.32.2 - - [18/Apr/2023:23:42:20 +0200] "GET /ws HTTP/1.1" 101 4049 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36" "192.168.1.1, 172.30.32.1"
2023-04-18 21:42:20.828213866  172.30.32.1 - - [18/Apr/2023:23:42:20 +0200] "GET /api/stats HTTP/1.1" 200 3000 "-" "HomeAssistant/2023.4.5 aiohttp/3.8.4 Python/3.10" "-"
2023-04-18 21:42:21.918629119  [2023-04-18 23:42:21] watchdog.doorbell              ERROR   : Ffmpeg process crashed unexpectedly for doorbell.
2023-04-18 21:42:21.921861721  [2023-04-18 23:42:21] watchdog.doorbell              ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
2023-04-18 21:42:21.922092845  [2023-04-18 23:42:21] ffmpeg.doorbell.detect         ERROR   : rtsp://127.0.0.1:8554/doorbell: Invalid data found when processing input
```

### FFprobe output from your camera

```shell
Input #0, rtsp, from 'rtsp://XXX:XXX@192.168.1.XXX:554/onvif1':
  Metadata:
    title           : H.264 Video, RtspServer_0.0.0.2
  Duration: N/A, start: 0.000000, bitrate: N/A
  Stream #0:0: Video: h264 (High), yuvj420p(pc, bt709, progressive), 1920x1080, 10 tbr, 90k tbn
  Stream #0:1: Audio: pcm_alaw, 8000 Hz, mono, s16, 64 kb/s
```

### Frigate stats

```json
{"cpu_usages":{"%Cpu(s):":{"cpu":"id,","mem":"2.4"},"1":{"cpu":"0.0","mem":"0.0"},"107":{"cpu":"0.0","mem":"0.0"},"109":{"cpu":"3.7","mem":"2.8"},"120":{"cpu":"0.0","mem":"0.2"},"134":{"cpu":"0.0","mem":"0.1"},"135":{"cpu":"0.0","mem":"0.1"},"136":{"cpu":"0.3","mem":"0.1"},"138":{"cpu":"0.0","mem":"0.1"},"15":{"cpu":"0.0","mem":"0.0"},"16":{"cpu":"0.0","mem":"0.0"},"24":{"cpu":"0.0","mem":"0.0"},"25":{"cpu":"0.0","mem":"0.0"},"26":{"cpu":"0.0","mem":"0.0"},"27":{"cpu":"0.0","mem":"0.0"},"28":{"cpu":"0.0","mem":"0.0"},"29":{"cpu":"0.0","mem":"0.0"},"291":{"cpu":"0.0","mem":"1.0"},"297":{"cpu":"0.7","mem":"0.1"},"298":{"cpu":"0.0","mem":"5.5"},"299":{"cpu":"3.0","mem":"1.2"},"30":{"cpu":"0.0","mem":"0.0"},"302":{"cpu":"0.0","mem":"1.3"},"304":{"cpu":"2.0","mem":"1.3"},"305":{"cpu":"1.0","mem":"1.2"},"306":{"cpu":"1.7","mem":"1.2"},"307":{"cpu":"0.0","mem":"1.2"},"308":{"cpu":"0.0","mem":"0.2"},"309":{"cpu":"9.3","mem":"1.2"},"31":{"cpu":"0.0","mem":"0.0"},"310":{"cpu":"2.0","mem":"1.2"},"313":{"cpu":"0.0","mem":"0.2"},"314":{"cpu":"8.6","mem":"1.3"},"32":{"cpu":"0.0","mem":"0.0"},"322":{"cpu":"9.6","mem":"0.9"},"323":{"cpu":"3.7","mem":"0.6"},"324":{"cpu":"0.0","mem":"0.2"},"328":{"cpu":"0.0","mem":"0.2"},"329":{"cpu":"10.0","mem":"0.9"},"332":{"cpu":"0.0","mem":"0.2"},"40":{"cpu":"0.0","mem":"0.0"},"42":{"cpu":"0.0","mem":"0.0"},"745":{"cpu":"0.0","mem":"0.0"},"78":{"cpu":"0.0","mem":"0.0"},"808":{"cpu":"0.3","mem":"0.0"},"81":{"cpu":"0.0","mem":"0.0"},"811":{"cpu":"0.0","mem":"0.0"},"812":{"cpu":"0.0","mem":"0.2"},"82":{"cpu":"0.0","mem":"0.0"},"89":{"cpu":"4.3","mem":"0.3"},"MiB":{"cpu":"4171.3","mem":"avail"},"PID":{"cpu":"%CPU","mem":"%MEM"},"Tasks:":{"cpu":"stopped,","mem":"0"},"top":{"cpu":"users,","mem":"load"}},"detection_fps":0.0,"detectors":{"ov":{"detection_start":0.0,"inference_speed":30.19,"pid":298}},"doorbell":{"camera_fps":0.0,"capture_pid":307,"detection_enabled":1,"detection_fps":0.0,"ffmpeg_pid":793,"pid":302,"process_fps":0.3,"skipped_fps":0.0},"garage":{"camera_fps":10.1,"capture_pid":309,"detection_enabled":1,"detection_fps":0.0,"ffmpeg_pid":329,"pid":304,"process_fps":10.1,"skipped_fps":0.0},"garden":{"camera_fps":10.0,"capture_pid":310,"detection_enabled":1,"detection_fps":0.0,"ffmpeg_pid":323,"pid":305,"process_fps":10.0,"skipped_fps":0.0},"gpu_usages":{"intel-vaapi":{"gpu":"6.43 %","mem":"- %"}},"right_room":{"camera_fps":10.1,"capture_pid":314,"detection_enabled":0,"detection_fps":0.0,"ffmpeg_pid":322,"pid":306,"process_fps":10.0,"skipped_fps":0.0},"service":{"last_updated":1681854033,"latest_version":"0.12.0","storage":{"/dev/shm":{"free":4093.2,"mount_type":"tmpfs","total":4104.8,"used":11.5},"/media/frigate/clips":{"free":271610.0,"mount_type":"ext4","total":491106.5,"used":194474.4},"/media/frigate/recordings":{"free":271610.0,"mount_type":"ext4","total":491106.5,"used":194474.4},"/tmp/cache":{"free":4104.7,"mount_type":"tmpfs","total":4104.8,"used":0.0}},"temperatures":{},"uptime":282,"version":"0.12.0-da3e197"}}
```

### Operating system

Debian

### Install method

HassOS Addon

### Coral version

CPU (no coral)

### Network connection

Wireless

### Camera make and model

Yoosee sd-m5

**Comment by NickM-27** [comment]
This is going to be an issue with go2rtc providing the rtsp stream and the webrtc stream at the same time. This has also been improved in go2rtc 1.3.x while frigate still runs the 1.2.0 version of go2rtc.

**Comment by Uriziel01** [comment]
@NickM-27  So could this be somewhat mitigated by using this binary sideloading from latest master branch of Frigate (https://github.com/blakeblackshear/frigate/commit/1b8cd10142b6db04d28ff1067809a8ab40315982) and just using 1.3.2 version of go2rtc?

**Comment by NickM-27** [comment]
The go2rtc side loading is included in 0.12 release. And you can try that, either way it's going to be a go2rtc issue.

**Comment by Uriziel01** [comment]
You just have to put go2rtc binary from github appropriate for you architecture in the `/config` directory of HA and name it exactly `go2rtc`, you can make sure it was loaded when after restarting Frigate you see such a message in the addon log `[WARN] Using go2rtc binary from '/config/go2rtc' instead of the embedded one`

**Comment by tmjpugh** [comment]
how sideload go2rtc when not using HA addon?

**Comment by NickM-27** [comment]
> how sideload go2rtc when not using HA addon?

exactly the same as @Uriziel01 described above
